.gitignore

```diff
@@ -18,4 +18,4 @@
 
 .byebug_history
 
-DS_Store
+~/.DS_Store
```

Comment:
This is a platform specific file, and as such would be better put in your .gitignore, and not the one in this repository.

If you really do want it in here, I think `.DS_Store` is the right thing to ignore, not `~/.DS_Store`, as I'm not sure what `~` will do in a .gitignore file, and you don't want the `/` as you want to ignore the files at all levels of the hierarchy.